.github/workflows/linux_meson.yml

```diff
@@ -198,3 +200,55 @@ jobs:
           cd doc
           python3-dbg -m pip install pytest pytest-xdist pytest-timeout mpmath gmpy2 threadpoolctl pooch
           python3-dbg -m pytest --pyargs scipy -n2 --durations=10 -m "not slow"
+
+  #################################################################################
+  gcc8:
+    # Purpose is to examine builds with oldest-supported gcc.
+    name: Build with gcc-8
+    if: "github.repository == 'scipy/scipy' || github.repository == ''"
+    runs-on: ubuntu-20.04  # 22.04 doesn't support gcc-8
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.9"
+
+      - name: Setup OS
+        run: |
+          sudo apt-get -y update
+          sudo apt install -y g++-8 gcc-8 gfortran-8
+          sudo apt install -y libatlas-base-dev liblapack-dev libgmp-dev \
+            libmpfr-dev libmpc-dev pkg-config libsuitesparse-dev
+
+          # install openblas
+          chmod +x tools/wheels/cibw_before_build_linux.sh
+          sudo tools/wheels/cibw_before_build_linux.sh .
+
+      - name: Setup Python deps
+        run: |
+          pip install --upgrade "numpy==1.21.6" &&
+          pip install --upgrade pip setuptools==59.6.0 wheel build meson meson-python ninja &&
```

Comment:
`setuptools` and `wheel` are no longer needed. So this like can be:
```
pip install build meson-python ninja pythran pybind11 "cython==0.29.34"
```
note that I moved `pythran`, `pybind11` and `cython` one line up, so that all the build deps are in this line, and all test deps in the next line.